modules/wallet/signtransaction_test.go

```diff
@@ -0,0 +1,56 @@
+package wallet
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// mockHashSigner serves as a mock replacement for crypto.SignHash.
+type mockHashSigner struct {
+	mock.Mock
+}
+
+func (hs *mockHashSigner) SignHash(hash crypto.Hash, key crypto.SecretKey) (sig crypto.Signature, err error) {
```

Comment:
looks like the error returned by the SignHash function is actually useless. Given that signing a hash can't actually return an error, mocking should not be required. I can make a PR that changes the function definition of SignHash.